lib/kafo/kafo_configure.rb

```diff
@@ -474,6 +474,7 @@ def run_installation
     end
 
     def progress_log(method, message)
+      return if method.nil? || message.nil?
```

Comment:
```suggestion
      return if method.nil? && message.nil?
```

I think this makes more sense given what you are doing at https://github.com/theforeman/kafo/pull/268/files#diff-95215de21e324444e4ed6b4a0ffd9f77R10

Reply:
I was worried a bit that if either were nil, the following code would bomb out. I think the code requires both to work from this point on. I have some further re-factoring ideas for all of this area but have been trying to tackle small little fixes first.

Reply:
@wbclark thoughts?